addons/portal/models/res_partner.py

```diff
@@ -18,3 +40,16 @@ def can_edit_vat(self):
         edit it (as in backend)."""
         self.ensure_one()
         return not self.parent_id
+
+    def can_edit_info(self):
+        """ Overide this method to allow user to change address information. """
+        return True
```

Comment:
I didn't find where this method is being overrided. Did I miss something? :thinking: 
```suggestion
```